doc/documentation_data_format.md

```diff
@@ -109,36 +109,47 @@ Any parameters named `noiseParameter${1..n}` *must* be overwritten in the
 
 ## Condition table
 
-The condition table specifies parameters or *constant* species for specific
+The condition table specifies parameters, initial values of species and compartments for specific
 simulation conditions (generally corresponding to different experimental
 conditions).
 
 This is specified as tab-separated value file with condition-specific
-species/parameters in the following way:
+species/parameters/compartments in the following way:
 
-| conditionId | [conditionName] | parameterOrStateId1 | ... | parameterOrStateId${n} |
+| conditionId | [conditionName] | parameterOrStateOrCompartmentId1 | ... | parameterOrStateOrCompartmentId${n} |
 |---|---|---|---|---|
-| conditionId1 | conditionName1 | NUMERIC&#124;parameterId | ...| ...
+| conditionId1 | conditionName1 | NUMERIC&#124;parameterId&#124;stateId&#124;compartmentId | ...| ...
 | conditionId2 | ... | ... | ...| ...
 |... | ... | ... | ... |...| ...
 
 Row names are condition names as referenced in the measurement table below.
-Column names are global parameter IDs or IDs of constant species as given in
-the SBML model. These parameters will override any parameter values specified
-in the model. `parameterOrStateId`s and `conditionId`s must be unique.
+Column names are global parameter IDs, IDs of species or compartments as given
+in the SBML model. These parameters will override any parameter values
+specified in the model. `parameterOrStateId`s and `conditionId`s must be
+unique.
+
+If a species ID is provided, it is interpreted as the initial
+concentration/amount of that species and will override the initial
+concentration/amount given in the SBML model or given by a preequilibration
+condition. If `NaN` is provided for a condition, the result of the
+preequilibration (or initial concentration/amount from the SBML model, if no
+preequilibration is defined) is used.
+
+If a compartment Id is provided, it is interpreted as the initial compartment
+size.
 
 Values for condition parameters may be provided either as numeric values, or
-as parameter IDs. In case parameter IDs are provided, they need to be defined
-in the SBML model, the parameter table or both. 
+as IDs defined in the SBML model, the parameter table or both.
```

Comment:
what would be a scenario where the id is only defined in the parameter table and it is to be overwritten in a condition-specific manner?

Reply:
You could set output parameters in a condition-specific manner, avoiding the not so aesthetic observableParameter1_observableId construct.